steps/sonarqube-msbuild-scanner/1.0.0/step.yml

```diff
@@ -0,0 +1,113 @@
+title: SonarQube MSBuild Scanner
+summary: |
+  Step for running SonarQube MSBuild Scanner CLI.
+description: |
+  Step for running SonarQube MSBuild Scanner CLI.
+website: https://github.com/tbrushwyler/bitrise-step-sonar-qube-msbuild-scanner
+source_code_url: https://github.com/tbrushwyler/bitrise-step-sonar-qube-msbuild-scanner
+support_url: https://github.com/tbrushwyler/bitrise-step-sonar-qube-msbuild-scanner/issues
+published_at: 2018-12-11T15:47:20.764047285-05:00
+source:
+  git: https://github.com/tbrushwyler/bitrise-step-sonarqube-msbuild-scanner.git
+  commit: 130fd867ccf98641413eb79703925cdfaa551f8d
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- xamarin
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: Scanner CLI version to be used. Step will fail if invalid or non-existent
+      is specified.
+    is_required: true
+    title: Scanner CLI version
+  scanner_version: 4.5.0.1761
+- opts:
+    description: Specifies the key of the analyzed project in SonarQube.
+    is_required: true
+    title: Key of the analyzed project in SonarQube.
+  sonarqube_project_key: null
+- opts:
+    description: Specifies the name of the analyzed project in SonarQube. Adding this
+      argument will overwrite the project name in SonarQube if it already exists.
+    is_required: false
+    title: Name of the analyzed project in SonarQube.
+  sonarqube_project_name: null
+- opts:
+    description: Specifies the version of your project.
+    is_required: false
+    title: Version of your project in SonarQube.
+  sonarqube_project_version: null
+- opts:
+    description: |-
+      See [analysis parameters](https://docs.sonarqube.org/display/SONAR/Analysis+Parameters) documentation
+      for list of possible options.
+
+      Example:
+      ```
+      sonar.sources=.
+      sonar.java.binaries=build/classes/kotlin/main
+      sonar.login=TOKEN_HERE
```

Comment:
If sensitive data is passed as input, `is_sensitive: true` property is recommended, to prevent logging of secret data (https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#secret-environment-variables-in-steps).